### PR TITLE
Feat/breadcrumbs

### PR DESCRIPTION
**Pull request by bobetbat**
- map zoom
- fixed breaking img fallback

### CODE REVIEW

File: src/containers/GuestInfoContainer.tsx [hunk 1]
```diff
@@ -58,7 +58,7 @@ export const GuestInfoContainer = () => {
 
   const methods = useForm<PersonalInfo>({
     resolver: yupResolver(NewUserSchema),
```
Comment (kostysh) [inline src/containers/GuestInfoContainer.tsx:60]: Why we are using the yup package for object validation?
Our glider-types package contains JSON schema for the Guest type, so we can validate objects using the schema. OK, if you do not like a schema validator helper from the `@windingtree/org.id-utils` package you always can use the original the `ajv` package.
If the original Guest info definition (from the win types) will be changed you will be required to refactor your yup code but  if  you will use JSON schema based validation you should not touch anything
If usage of yup is mandatory for the useForm hook you have to use types from the schema at least in the yup resolver definition.

File: src/pages/Checkout.tsx [hunk 2]
```diff
@@ -27,6 +29,12 @@ export const Checkout = () => {
   const theme = useTheme();
   const isDesktop = useResponsive('up', 'md');
   const { checkout, account } = useAppState();
+  const { latestQueryParams } = useAccommodationsAndOffers();
+  const query = useMemo(
+    () => createSearchParams(JSON.stringify(latestQueryParams)),
```
Comment (kostysh) [inline src/pages/Checkout.tsx:34]: `latestQueryParams` can be `undefined`. `JSON.stringify(latestQueryParams)` will produce a string `undefined`.
Is it OK to pass "undefined" to the `createSearchParams` utility?

File: src/containers/GuestInfoContainer.tsx (hunk 1 above)
Comment (bobetbat) [reply src/containers/GuestInfoContainer.tsx:60]: Can we leave it to refactor pr?

Comment (kostysh) [reply src/containers/GuestInfoContainer.tsx:60]: Do not change it for now.

**Review by kostysh — APPROVED** [review]
LGTM

**Pull request merged**